pkg/genyaml/testdata/omit_if_empty/example_config.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package omit
+
+type OmitEmptyStrings struct {
+	// StringField comment
+	StringField string `json:"string,omitempty"`
```

Comment:
It would be nice to see an `omitempty` test that is _not_ omitted (did I miss one?)

Reply:
I added an additional [field](https://github.com/kubernetes/test-infra/pull/13637/commits/5b18160d003f4e0954ca7ebddbc10240f62f9633#diff-d14ff2031c62c35379c2316b20fda6ecR378) to the [test](https://github.com/kubernetes/test-infra/pull/13637/commits/5b18160d003f4e0954ca7ebddbc10240f62f9633#diff-d14ff2031c62c35379c2316b20fda6ecR378) which shows preservation of the field when `omitifempty` is **excluded**